libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -76,20 +79,29 @@ const getManagedNetworkingDisabledReason = (
 };
 
 const getUserManagedDisabledReason = (
-  featureSupportLevelData: NewFeatureSupportLevelData,
+  featureSupportLevelContext: NewFeatureSupportLevelData,
   platformType?: PlatformType,
+  featureSupportLevelMap?: NewFeatureSupportLevelMap | null,
 ) => {
-  if (platformType === 'nutanix' || platformType === 'vsphere') {
-    return 'User-Managed Networking is not supported when using Nutanix or vSphere';
-  }
-  if (!featureSupportLevelData.isFeatureSupported('USER_MANAGED_NETWORKING')) {
-    return featureSupportLevelData.getFeatureDisabledReason('USER_MANAGED_NETWORKING');
+  if (!featureSupportLevelContext.isFeatureSupported('USER_MANAGED_NETWORKING')) {
+    return featureSupportLevelContext.getFeatureDisabledReason(
+      'USER_MANAGED_NETWORKING',
+      featureSupportLevelMap ?? undefined,
```

Comment:
this looks weird...

Reply:
is there any difference when featureSupportLevelMap is null or undefined?

Reply:
NVM, let's try to improve this as part of the refactor we plan to do around the FeatureSupportLevelsProvider (or something like that)

Reply:
We have a ticket to do the refactor of FeatureSupportLevelsProvider and use redux.